homeassistant/components/unifi/unifi_entity_base.py

```diff
@@ -9,11 +9,13 @@
 class UniFiBase(Entity):
     """UniFi entity base class."""
 
+    DOMAIN = ""
     TYPE = ""
 
     def __init__(self, controller) -> None:
         """Set up UniFi entity base."""
         self.controller = controller
+        self.controller.entities[self.DOMAIN][self.TYPE].add(self.mac)
```

Comment:
Add a comment why it's important to add the mac here and not in `async_added_to_hass`.